### PR TITLE
Change census-instrumentation to open-telemetry and update authors

### PR DESCRIPTION
**Pull request by flands**
census-instrumentation/opencensus-service is now open-telemetry/opentelemetry-service

"OpenCensus Authors" is now "OpenTelemetry Authors"

"Copyright 2018" is now "Copyright 2019"

### CODE REVIEW

File: exporter/zipkinexporter/zipkin_test.go [hunk 1]
```diff
@@ -28,14 +28,14 @@ import (
 	"strings"
 	"testing"
 
-	commonpb "github.com/census-instrumentation/opencensus-proto/gen-go/agent/common/v1"
+	commonpb "github.com/open-telemtery/opentelemetry-proto/gen-go/agent/common/v1"
```
Comment (tigrannajaryan) [inline exporter/zipkinexporter/zipkin_test.go]: I think this won't work, there is no `open-telemtery/opentelemetry-proto`. This should stay with opencensus.
Same everywhere else opentelemetry-proto is referenced.

File: exporter/zipkinexporter/zipkin.go [hunk 2]
```diff
@@ -228,14 +228,14 @@ func (ze *zipkinExporter) ConsumeTraceData(ctx context.Context, td data.TraceDat
 }
 
 // This code from down below is mostly copied from
-// https://github.com/census-instrumentation/opencensus-go/blob/96e75b88df843315da521168a0e3b11792088728/exporter/zipkin/zipkin.go#L57-L194
+// https://github.com/open-telemtery/opentelemetry-go/blob/96e75b88df843315da521168a0e3b11792088728/exporter/zipkin/zipkin.go#L57-L194
```
Comment (tigrannajaryan) [inline exporter/zipkinexporter/zipkin.go]: The new link is valid, please keep the old link.

File: exporter/zipkinexporter/zipkin.go [hunk 3]
```diff
@@ -228,14 +228,14 @@ func (ze *zipkinExporter) ConsumeTraceData(ctx context.Context, td data.TraceDat
...
 // but that is because the Zipkin Go exporter requires process to change
 // and was designed without taking into account that LocalEndpoint and RemoteEndpoint
 // are per-span-Node attributes instead of global/system variables.
 // The alternative is to create a single exporter for every single combination
 // but this wastes resources i.e. an HTTP client for every single combination
 // but also requires the exporter to be changed entirely as per
-// https://github.com/census-instrumentation/opencensus-go/issues/959
+// https://github.com/open-telemtery/opentelemetry-go/issues/959
```
Comment (tigrannajaryan) [inline exporter/zipkinexporter/zipkin.go]: Keep the old link.

File: go.mod [hunk 4]
```diff
@@ -10,7 +10,7 @@ require (
 	github.com/apache/thrift v0.0.0-20161221203622-b2a4d4ae21c7
 	github.com/aws/aws-sdk-go v1.19.18 // indirect
 	github.com/bmizerany/perks v0.0.0-20141205001514-d9a9656a3a4b // indirect
-	github.com/census-instrumentation/opencensus-proto v0.2.0
+	github.com/open-telemtery/opentelemetry-proto v0.2.0
```
Comment (tigrannajaryan) [inline go.mod]: Same here.

File: internal/collector/opencensus/receiver_test.go [hunk 5]
```diff
@@ -93,7 +93,7 @@ func TestStart(t *testing.T) {
 			}
 			if got != nil {
 				// TODO: (@pjanotti) current StopTraceReception, stop the whole receiver.
-				// See https://github.com/census-instrumentation/opencensus-service/issues/559
+				// See https://github.com/open-telemtery/opentelemetry-service/issues/559
```
Comment (tigrannajaryan) [inline internal/collector/opencensus/receiver_test.go]: Keep links to issues unchanged.

File: docs/migrating-from-opencensus.md [hunk 6]
```diff
@@ -9,7 +9,7 @@ The contrib package and vendor-specific receivers and exporters will continue to
 
 This is the action plan that also shows the progress. Tick the boxes after the task is complete.
 
-[ ] Copy all commits from https://github.com/census-instrumentation/opencensus-service to https://github.com/open-telemetry/opentelemetry-service
+[ ] Copy all commits from https://github.com/open-telemetry/opentelemetry-service to https://github.com/open-telemetry/opentelemetry-service
```
Comment (pjanotti) [inline docs/migrating-from-opencensus.md]: No renames on this file plese.

File: docs/migrating-from-opencensus.md [hunk 7]
```diff
@@ -35,7 +35,7 @@ The functionally of the new `otsvc` will heavily lean on existing implementation
 
 [ ] Provide guidelines and example implementations for vendors to follow when they add new receivers and exporters to the contrib package.
 
-[ ] Create a new repository for contrib and copy all commits from https://github.com/census-instrumentation/opencensus-service to https://github.com/open-telemetry/opentelemetry-service
+[ ] Create a new repository for contrib and copy all commits from https://github.com/open-telemetry/opentelemetry-service to https://github.com/open-telemetry/opentelemetry-service
```
Comment (songy23) [inline docs/migrating-from-opencensus.md]: Same for this one - should be `opencensus`.

File: internal/config/config.go [hunk 8]
```diff
@@ -26,14 +26,14 @@ import (
...
-	"github.com/census-instrumentation/opencensus-service/exporter/jaegerexporter"
-	"github.com/census-instrumentation/opencensus-service/exporter/kafkaexporter"
-	"github.com/census-instrumentation/opencensus-service/exporter/opencensusexporter"
-	"github.com/census-instrumentation/opencensus-service/exporter/prometheusexporter"
-	"github.com/census-instrumentation/opencensus-service/exporter/zipkinexporter"
-	"github.com/census-instrumentation/opencensus-service/receiver/opencensusreceiver"
-	"github.com/census-instrumentation/opencensus-service/receiver/prometheusreceiver"
+	"github.com/open-telemtery/opentelemetry-service/consumer"
+	"github.com/open-telemtery/opentelemetry-service/exporter/jaegerexporter"
+	"github.com/open-telemtery/opentelemetry-service/exporter/kafkaexporter"
```
Comment (tigrannajaryan) [inline internal/config/config.go]: Can you also remove kafka exporter?

**Pull request merged**